test/integration/content-explorer/ContentExplorer.e2e.test.js

```diff
@@ -32,5 +32,49 @@ describe('ContentExplorer', () => {
             cy.getByTitle('Previous File').click();
             cy.getByTestId('bcs-content').should('not.exist');
         });
+
+        // precedence order: skills > activity > details > metadata
+        it('Navigating from file to file, tabs should switch based on order of precedence', () => {
+            // Sidebar should be open by default
+            cy.getByTestId('bcs-content').should('exist');
+            // Skills tab should be selected
+            cy.getByTestId('sidebarskills').should('have.class', 'bcs-nav-btn-is-selected');
+
+            // Navigate to previous file should default to activity feed
```

Comment:
Is this because the previous file is a specific file or type of file? If so, it might be good to add an assertion that we're looking at the expected file name.

Reply:
Yeah, i'll add that assertion 👍